lib/gltfPipeline.js

```diff
@@ -52,6 +53,13 @@ function writeSources(gltf) {
         });
 }
 
+/**
+ * Add pipeline extras and load uris, then process the gltf.
+ *
+ * @param {Object} gltf A javascript object holding a glTF hierarchy.
+ * @param {Object} options Options to apply to stages during optimization.
```

Comment:
Well it may be redundant in some cases. Maybe just make a note to refer to the `processJSONWithExtras` doc.